src/main/java/teammates/common/datatransfer/attributes/NotificationAttributes.java

```diff
@@ -140,6 +144,10 @@ public void setShown() {
     }
 
     public Instant getCreatedAt() {
+        // TODO: temporary fix for null createdAt
```

Comment:
Was this for testing? Not sure if createdAt will be null

Reply:
Before this PR, the POST request actually creates entities with null createdAt. So this was to view those entities.